linux/utils/check_guest_os_gui.yml

```diff
@@ -1,16 +1,63 @@
 # Copyright 2021-2022 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
 ---
+# Check whether guest OS has desktop environment and
+# get display manager if deskotp environment is installed"
+- name: "Set fact of guest OS having desktop environment and display manager"
```

Comment:
Task name can be changed to "Initialize" variables

Reply:
Updated